### PR TITLE
Fancy-fy README.md. No code changed.

### PR DESCRIPTION
**Pull request by tomdrever**

### CODE REVIEW

**Comment by tomdrever** [comment]
I presumed "pdf.js" was the mozilla one...

**Comment by tomdrever** [comment]
Mind the lower case "s" on #REMINDERs. Missed it.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+#Past papers#
+####An application that fires random questions from past exam papers at you.####
 
-these questions can be controlled by simple filters.
-I am working on GCSE level first, then a level, help will be needed to add other states which I do not know about.
+These questions can be controlled by simple filters.
+I am working on GCSE level first, then A level. Help will be needed in adding other levels of education.
 
-The program has a backend which does the processing of questions via regex and will have a electron frontend gui as I have had difficulties building electron in windows.
+The program has a backend which does the processing of questions, using regex, and will have an [electron](http://electron.atom.io/) frontend gui, via electron, due to difficulties building electron in windows.
```
Comment (jamierocks) [inline README.md:7]: This whole sentence makes no sense.

**Comment by UnkDevE** [comment]
yup me was too lazy to delete all stuffs srry

**Pull request merged**